package.json

```diff
@@ -54,24 +54,24 @@
     "yargs": "11.0.0"
   },
   "scripts": {
-    "build": "./node_modules/.bin/gulp build",
-    "clean": "./node_modules/.bin/gulp clean",
-    "deploy": "./node_modules/.bin/gulp deploy",
-    "promote": "./node_modules/.bin/gulp promote",
-    "burgr": "./node_modules/.bin/gulp burgr:sonarcloud && ./node_modules/.bin/gulp burgr:sonarqube",
-    "sonarqube": "./node_modules/.bin/gulp sonarqube-analysis:sonarqube && ./node_modules/.bin/gulp sonarqube-analysis:sonarcloud",
-    "format": "./node_modules/.bin/prettier --write --list-different \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "format-check": "./node_modules/.bin/prettier --list-different \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "install-dep": "./node_modules/.bin/gulp npminstall",
-    "install-depv4": "./node_modules/.bin/gulp npminstallv4",
-    "install-depv5": "./node_modules/.bin/gulp npminstallv5",
-    "install-dep-full": "./node_modules/.bin/gulp npminstall && ./node_modules/.bin/gulp npminstallv4 && ./node_modules/.bin/gulp npminstallv5 && npm run clean",
+    "build": "gulp build",
+    "clean": "gulp clean",
+    "deploy": "gulp deploy",
+    "promote": "gulp promote",
+    "burgr": "gulp burgr:sonarcloud && gulp burgr:sonarqube",
+    "sonarqube": "gulp sonarqube-analysis:sonarqube && gulp sonarqube-analysis:sonarcloud",
+    "format": "prettier --write --list-different \"{commonv5,common,extensions}/**/*.{js,ts}\"",
+    "format-check": "prettier --list-different \"{commonv5,common,extensions}/**/*.{js,ts}\"",
+    "install-dep": "npm i && gulp npminstall",
+    "install-depv4": "npm i && gulp npminstallv4",
+    "install-depv5": "npm i && gulp npminstallv5",
+    "install-dep-full": "npm i && gulp npminstall && gulp npminstallv4 && gulp npminstallv5 && npm run clean",
     "lint": "eslint --ext js,ts --quiet \"{commonv5,common,extensions}/**/*.{js,ts}\"",
     "lint-report": "eslint --ext js,ts -f json -o eslint-report.json \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "test-build": "./node_modules/.bin/gulp build:test",
-    "test": "./node_modules/.bin/jest",
-    "test-ci": "./node_modules/.bin/jest --coverage",
-    "ts-check": "./node_modules/.bin/tsc --noEmit",
+    "test-build": "gulp build:test",
+    "test": "jest",
+    "test-ci": "jest --coverage",
+    "ts-check": "tsc --noEmit",
```

Comment:
Removed `./node_modules/.bin/` prefix since AFAIK, npm already adds it to the PATH before executing any of these scripts